docs/LangSpec.md

```diff
@@ -100,32 +100,32 @@ As in many languages Array is a collection of items of the same type.
 
 Each item can be accessed by constant index, for example:
 ```
-let myOutput = OUTPUTS(0)
-let myInput = INPUTS(0)
+val myOutput = OUTPUTS(0)
+val myInput = INPUTS(0)
 ```
 
 Array have `size` property which returns number of elements in an array. 
 
 ```
-let size = OUTPUTS.size
+val size = OUTPUTS.size
 ```
 
 Even though ErgoScript is not object-oriented language, Array operations use the syntax of method invocations. 
 For example the following script check an existence of some element in the array satisfying some predicate (condition)
 
 ```
-let ok = OUTPUTS.exists(fun (box: Box) = box.value > 1000)
+val ok = OUTPUTS.exists({ (box: Box) => box.value > 1000 })
```

Comment:
@greenhat do we support `OUTPUTS.exists { (box: Box) => box.value > 1000 }` ?

Reply:
@ergomorphic Totally forgot and did not even try it. Let me check.

Reply:
@ergomorphic Yes, `OUTPUTS.exists { (box: Box) => box.value > 1000 }` works. I've used this syntax in some tests, wpaper and language spec.
I've tried to implement `OUTPUTS.exists ( (box: Box) => box.value > 1000 )` with no luck.

Reply:
@ergomorphic Done.